### ISSUE TITLE
upstream upgrade with --airgap-bundle and --deploy doesn't deploy

### ISSUE THREAD

**Issue by MikaelSmith** [replicatedhq/kots#1200]
Running `kubectl kots upstream upgrade <app> --airgap-bundle <app-release>.airgap --kotsadm-namespace <app> --namespace default --deploy` in an airgap kURL cluster with KOTS 1.20.2 doesn't deploy the bundle.

It uploads all the updates and stages the new version. CLI output ends with
```
...
Writing manifest to image destination
Storing signatures
  • Checking for application updates ✓

  • There are no application updates available, ensuring latest is marked as deployed

  • To access the Admin Console, run kubectl kots admin-console --namespace default
```

However after that nothing happens. There are no log updates in any of the kotsadm pods. The app is not deployed.
<img width="1073" alt="Screen Shot 2020-10-29 at 3 21 11 PM" src="https://user-images.githubusercontent.com/8162496/97639036-08cbed00-19fb-11eb-9582-f7516bd885cc.png">

**Comment by MikaelSmith** [comment]
This is necessary to complete the headless offline workflow that https://kots.io/kotsadm/installing/installing-embedded-cluster/#airgapped-installations talks about. It kind of seems like `--deploy` and `--airgap-bundle` weren't really wired up and tested together.

**Comment by MikaelSmith** [comment]
This was marked as fixed in 1.23.0. Looks like https://github.com/replicatedhq/kots/pull/1238 addressed it.
